<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewProjectDialog</class>
 <widget class="QWizard" name="NewProjectDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Novo Projeto</string>
  </property>
  <property name="locale">
   <locale language="Portuguese" country="Brazil"/>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <property name="wizardStyle">
   <enum>QWizard::ModernStyle</enum>
  </property>
  <property name="options">
   <set>QWizard::NoDefaultButton</set>
  </property>
  <widget class="QWizardPage" name="wizardPage1">
   <property name="title">
    <string>Novo Projeto</string>
   </property>
   <property name="subTitle">
    <string>Configure nome e localização do projeto:</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>145</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="projectNameLabel">
        <property name="text">
         <string>Nome do Projeto:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="folderLabel">
        <property name="text">
         <string>Diretório:</string>
        </property>
        <property name="buddy">
         <cstring>folderEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="folderEdit"/>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="searchButton">
        <property name="text">
         <string>Procurar</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="projectNameEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>145</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
   <zorder>verticalSpacer_2</zorder>
   <zorder>verticalSpacer_3</zorder>
  </widget>
  <widget class="QWizardPage" name="wizardPage2">
   <property name="title">
    <string>Hierarquia de tipos</string>
   </property>
   <property name="subTitle">
    <string>Defina a hierarquia dos tipos para ligações entre cláusulas:</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QTreeWidget" name="TIMTreeWidget">
      <column>
       <property name="text">
        <string>TIM</string>
       </property>
      </column>
      <item>
       <property name="text">
        <string>&lt;adicionar tipo&gt;</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>A hierarquia irá definir o TIM (Traceability Model Information)</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPage3">
   <property name="title">
    <string>Novo projeto</string>
   </property>
   <property name="subTitle">
    <string>Resumo do novo projeto:</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="projectNameLabel_2">
        <property name="text">
         <string>Nome do Projeto:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="locationLabel">
        <property name="text">
         <string>Localização:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="locationForm">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="projectForm">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>TIM (Traceability Information Model)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="interactive">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>projectNameEdit</tabstop>
  <tabstop>folderEdit</tabstop>
  <tabstop>searchButton</tabstop>
  <tabstop>TIMTreeWidget</tabstop>
  <tabstop>graphicsView</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>projectNameEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>projectForm</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>242</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>folderEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>locationForm</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>TIMTreeWidget</sender>
   <signal>itemChanged(QTreeWidgetItem*,int)</signal>
   <receiver>graphicsView</receiver>
   <slot>invalidateScene()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>287</x>
     <y>267</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>314</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
